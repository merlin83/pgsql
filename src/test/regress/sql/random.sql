--
-- test the random function
--
-- count the number of tuples originally
SELECT count(*) FROM onek;

-- select roughly 1/10 of the tuples
SELECT count(*) FROM onek where oidrand(onek.oid, 10);

-- select again, the count should be different
SELECT count(*) FROM onek where oidrand(onek.oid, 10);

