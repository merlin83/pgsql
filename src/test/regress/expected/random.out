QUERY: SELECT count(*) FROM onek;
count
-----
 1000
(1 row)

QUERY: SELECT count(*) FROM onek where oidrand(onek.oid, 10);
count
-----
   92
(1 row)

QUERY: SELECT count(*) FROM onek where oidrand(onek.oid, 10);
count
-----
   98
(1 row)

